packages/mobile/src/transactions/TransactionsList.tsx

```diff
@@ -0,0 +1,291 @@
+import BigNumber from 'bignumber.js'
+import gql from 'graphql-tag'
+import * as React from 'react'
+import { Query } from 'react-apollo'
+import { connect } from 'react-redux'
+import {
+  MoneyAmount,
+  Token,
+  TransactionType,
+  UserTransactionsQuery,
+  UserTransactionsQueryVariables,
+} from 'src/apollo/types'
+import { CURRENCIES, CURRENCY_ENUM } from 'src/geth/consts'
+import { LocalCurrencyCode } from 'src/localCurrency/consts'
+import { getLocalCurrencyCode, getLocalCurrencyExchangeRate } from 'src/localCurrency/selectors'
+import { RootState } from 'src/redux/reducers'
+import { removeStandbyTransaction } from 'src/transactions/actions'
+import {
+  ExchangeStandby,
+  StandbyTransaction,
+  TransactionStatus,
+  TransferStandby,
+} from 'src/transactions/reducer'
+import TransactionFeed, { FeedItem, FeedType } from 'src/transactions/TransactionFeed'
+import { currentAccountSelector } from 'src/web3/selectors'
+
+interface OwnProps {
+  currency: CURRENCY_ENUM
+}
+
+interface StateProps {
+  address?: string | null
+  standbyTransactions: StandbyTransaction[]
+  localCurrencyCode: LocalCurrencyCode | null
+  localCurrencyExchangeRate: string | null | undefined
+}
+
+interface DispatchProps {
+  removeStandbyTransaction: typeof removeStandbyTransaction
+}
+
+type Props = OwnProps & StateProps & DispatchProps
+
+// See https://github.com/microsoft/TypeScript/issues/16069#issuecomment-565658443
+function isPresent<T>(t: T | undefined | null | void): t is T {
+  return t !== undefined && t !== null
+}
+
+export const TRANSACTIONS_QUERY = gql`
+  query UserTransactions($address: Address!, $token: Token!, $localCurrencyCode: String) {
+    transactions(address: $address, token: $token, localCurrencyCode: $localCurrencyCode) {
+      edges {
+        node {
+          ...TransactionFeed
+        }
+      }
+    }
+  }
+
+  ${TransactionFeed.fragments.transaction}
+`
+
+class UserTransactionsComponent extends Query<
+  UserTransactionsQuery,
+  UserTransactionsQueryVariables
+> {}
+
+const mapStateToProps = (state: RootState): StateProps => ({
+  address: currentAccountSelector(state),
+  standbyTransactions: state.transactions.standbyTransactions,
```

Comment:
what is the convention about selectors, do we create them only when some logic is involved to encapsulate it?

Reply:
I usually introduce them when it's need in more than 2 places. Or If I need reselect.